modules/starboard/handleStarboard.js

```diff
@@ -0,0 +1,99 @@
+const { embedTypeV2, disableModule, formatDiscordUserName } = require('../../src/functions/helpers');
+const { localize } = require('../../src/functions/localize');
+const { Op } = require('sequelize');
+
+module.exports = async (client, msgReaction, user, isReactionRemove = false) => {
+    if (!client.botReadyAt) return;
+    const msg = msgReaction.message;
+    if (!msg.guild) return;
+    if (msg.guild.id !== client.guildID) return;
+    if (msgReaction.partial) msgReaction = await msgReaction.fetch();
+
+    const starConfig = client.configurations['starboard']['config'];
+    if (!starConfig || (starConfig.emoji !== msgReaction.emoji.name && starConfig.emoji !== msgReaction.emoji.id)) return;
+    if (isNaN(starConfig.minStars)) return disableModule('starboard', localize('starboard', 'invalid-minstars', {stars: starConfig.minStars}));
+
+    const channel = client.channels.cache.get(starConfig.channelId);
+    if (!channel) return disableModule('starboard', localize('partner-list', 'channel-not-found', {c: starConfig.channelId}));
+    if ((msg.channel.nsfw && !channel.nsfw) || starConfig.excludedChannels.includes(msg.channel.id) || starConfig.excludedRoles.some(r => msg.member.roles.cache.has(r))) return;
+    if (!starConfig.selfStar && user.id === msg.author.id) return msgReaction.users.remove(user.id).catch(() => {});
+
+    const starUser = await client.models['starboard']['StarUser'].findAll({
+        where: {
+            userId: user.id,
+            createdAt: {
+                [Op.gt]: Date.now() - 1000 * 60 * 60
+            }
+        }
+    });
+
+    if (!isReactionRemove) {
+        if (starUser.length >= starConfig.starsPerHour) {
```

Comment:
```suggestion
        if (!isReactionRemove && starUser.length >= starConfig.starsPerHour) {
```
just make it one :wink: